public/script.js

```diff
@@ -19,77 +19,74 @@ const GROUPS = ["<=39", "40-44", "45-49", "50-54", "55-59", "60-64", "65-69", "7
 const GOVERNMENT_TARGET_NUM = 44_000_000;
 const GOVERNMENT_TARGET_PERCENT = GOVERNMENT_TARGET_NUM / 56_286_961;
 
-async function start() {
-    let latestDataDate = await getLatestDataDate();
-    await initializeBarCharts();
-    await initializeLineCharts(latestDataDate);
+function start() {
+    getLatestDataDate().then(initializeLineCharts);
+    initializeBarCharts();
 }
 
-async function initializeBarCharts() {
-    let csv = await d3.csv("latest.csv");
-    makeBarChart(
-        "bar-all",
-        "Percent of England vaccinated",
-        csv.filter(function(row) {
-            return row.group == "all";
-        }),
-        [
-            herdImmunityAnnotation("vertical", "x", true),
-            governmentTargetAnnotation("vertical", "x", true),
-        ],
-        false /* showGroups */);
-    makeBarChart(
-        "bar-over-80",
-        "Percent of >80s" + SUPERSCRIPT_4 + " vaccinated",
-        csv,
-        [],
-        true /* showGroups */);
+function initializeBarCharts() {
+    return d3.csv("latest.csv")
+        .then(csv => {
+            makeBarChart(
+                "bar-all",
+                "Percent of England vaccinated",
+                csv.filter(row => row.group === "all"),
+                [
+                    herdImmunityAnnotation("vertical", "x", true),
+                    governmentTargetAnnotation("vertical", "x", true)
+                ],
+                false /* showGroups */);
+            makeBarChart(
+                "bar-over-80",
+                "Percent of >80s" + SUPERSCRIPT_4 + " vaccinated",
+                csv,
+                [],
+                true /* showGroups */);
+        });
 }
 
 function makeBarChart(id, title, csv, annotations, showGroups) {
-    let vaccinated_per_dose = DOSES.map(function(dose) {
+    const vaccinated_per_dose = DOSES.map(dose => {
         // TODO: Unnest function.
-        let vaccinated = csv
-            .filter(function(row) {
+        const vaccinated = csv
+            .filter(row => {
                 if (showGroups && GROUPS.indexOf(row.group) === -1) {
                     return false;
                 }
                 if (!showGroups && row.group !== "all") {
                     return false;
                 }
-                return row.dose == dose;
+                return row.dose === dose;
             })
-            .map(function(row) {
-                let vaccinated = parseInt(row.vaccinated);
-                let population = parseFloat(row.population);
+            .map(row => {
+                const vaccinated = parseInt(row.vaccinated);
+                const population = parseFloat(row.population);
                 return {
                     x: (vaccinated / population) * 100,
                     vaccinated: vaccinated,
                     population: population,
-                    group: row.group,
+                    group: row.group
```

Comment:
IMO always trailing commas. Many languages enforce it, even. It makes git diffs smaller most of the time

Reply:
Proper solution is to add eslint hooks or whatever for formatting, so don't worry about any formatting nits.